libs/core/langchain_core/language_models/output_format.py

```diff
@@ -0,0 +1,18 @@
+from abc import ABC, abstractmethod
+from typing import Any, Generic, TypeVar
+
+from langchain_core.language_models import LanguageModelInput
+from langchain_core.runnables import Runnable
+
+_OutputSchema = TypeVar("_OutputSchema")
+_FormattedOutput = TypeVar("_FormattedOutput")
```

Comment:
Same q if we should restrict these a bit / somehow infer them based off each other. Seems a bit difficult to define FormattedOutput currently